pdns/recursordist/docs/metrics.rst

```diff
@@ -8,19 +8,43 @@ Regular Statistics Log
 Every half hour or so (configurable with :ref:`setting-statistics-interval`, the recursor outputs a line with statistics.
 To force the output of statistics, send the process a SIGUSR1. A line of statistics looks like this::
 
-    Feb 10 14:16:03 stats: 125784 questions, 13971 cache entries, 309 negative entries, 84% cache hits, outpacket/query ratio 37%, 12% throttled
+  stats: 346362 questions, 7388 cache entries, 1773 negative entries, 18% cache hits
+  stats: cache contended/acquired 1583/56041728 = 0.00282468%
+  stats: throttle map: 3, ns speeds: 1487, failed ns: 15, ednsmap: 1363
+  stats: outpacket/query ratio 54%, 0% throttled, 0 no-delegation drops
+  stats: 217 outgoing tcp connections, 0 queries running, 9155 outgoing timeouts
+  stats: 4536 packet cache entries, 82% packet cache hits
+  stats: thread 0 has been distributed 175728 queries
+  stats: thread 1 has been distributed 169484 queries
+  stats: 1 qps (average over 1800 seconds)
 
-This means that there are 13791 different names cached, which each may have multiple records attached to them.
-There are 309 items in the negative cache, items of which it is known that don't exist and won't do so for the near future.
-84% of incoming questions could be answered without any additional queries going out to the net.
+This means that in total 346362 queries were received and there are 7388 different name/type combinations in the record cache, each entry may have multiple records attached to it.
 
-The outpacket/query ratio means that on average, 0.37 packets were needed to answer a question.
-Initially this ratio may be well over 100% as additional queries may be needed to actually recurse the DNS and figure out the addresses of nameservers.
+There are 1773 items in the negative cache, items of which it is known that don't exist and won't do so for the near future.
+18% of incoming questions not handled by the packets cache could be answered without any additional queries going out to the net.
+The record cache was consulted or modified 56041728 times, and 1583 of those accesses caused lock contention.
 
-Finally, 12% of queries were not performed because identical queries had gone out previously and failed, saving load on servers worldwide.
+Next a line with the sizes of maps that can be consulted by :program:`rec_control` is printed.
+
+The outpacket/query ratio means that on average, 0.54 packets were needed to answer a question.
+This ration can be greater than 100% since additional queries could be needed to actually recurse the DNS and figure out the addresses of nameservers.
```

Comment:
```suggestion
This ratio can be greater than 100% since additional queries could be needed to actually recurse the DNS and figure out the addresses of nameservers.
```